diff --git a/sendHappy.php b/sendHappy.php
index 420f9c8..9534b3a 100644
--- a/sendHappy.php
+++ b/sendHappy.php
@@ -10,7 +10,14 @@ use Pimple\Container;
 
 $container = new Container();
 
-$emailLoader = new EmailAddressLoader();
-$happyMessageSender = new HappyMessageSender($emailLoader);
+$container['email_loader'] = function() {
+    return new EmailAddressLoader();
+};
+$container['happy_sender'] = function(Container $container) {
+    return new HappyMessageSender($container['email_loader']);
+};
+
+/** @var HappyMessageSender $happyMessageSender */
+$happyMessageSender = $container['happy_sender'];
 
 $happyMessageSender->sendHappiness();
